axelrod/tests/strategies/test_memory_decay.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for the Memory Decay strategy"""
+
+import axelrod as axe
+from axelrod.tests.strategies.test_player import TestPlayer
+
+C, D = axe.Action.C, axe.Action.D
+
+class TestMemoryDecay(TestPlayer):
+
+    name = 'MemoryDecay'
+    player = axe.meta.MemoryDecay
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        #Test TitForTat behavior in first 15 turns
+        opponent = axe.Cooperator()
+        actions = list([(C, C)]) * 15
+        self.versus_test(opponent, expected_actions = actions)
```

Comment:
PEP8 require no spaces around the `=` in function calls:

```
self.versus_test(opponent, expected_actions=actions)
```

(Throughout this file.)